File: agent/bindings/csharp/DebugMessage.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class DebugMessage
  {
    [JsonProperty(PropertyName = "channel")]
    public string Channel { get; set; }
    [JsonProperty(PropertyName = "message")]
    public string Message { get; set; }
  }
}

File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/DebugMessage.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class DebugMessage(
  val channel: String,
  val message: String,
  val level: DebugMessageLogLevel? = null, // Oneof: trace, debug, info, warn, error
)

